packages/jmix-react-ui/src/ui/MasterDetail/MasterDetail.tsx

```diff
@@ -0,0 +1,56 @@
+import { Row, Col, Card } from 'antd';
+import { observer } from 'mobx-react';
+import React from 'react';
+import {MasterDetailProvider, useMasterDetail} from './MasterDetailContext';
+
+interface MasterDetailProps {
+    editor: React.ReactNode;
+    browser: React.ReactNode;
+}
+
+const MasterDetailBrowserWrapper = observer(({children}: {children: React.ReactNode}) => {
+    const masterDetailStore = useMasterDetail();
+    
+    return (
+        <Card style={{height: '100%'}}>
+            {children}
+            {masterDetailStore.activeScreen !== 'browser' && (
+                <div
+                    style={{
+                        top: 0,
+                        left: 0,
+                        right: 0,
+                        bottom: 0,
+                        position: 'absolute',
+                        background: 'rgba(0,0,0,.1)',
+                    }}
+                />
+            )}
```

Comment:
I can't launch the app on Linux, so I can't see what exactly this will create, but I suppose you are graying-out the inactive component. I think it's better to do it in more straightforward way, perhaps to give the browser component a prop `inactive` and adjust the styling based on that from the component itself.